xbmc/platform/linux/AppParamParserWebOS.h

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright (C) 2005-2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "AppParamParserLinux.h"
+
+class CAppParamParserWebOS : public CAppParamParserLinux
+{
+public:
+  CAppParamParserWebOS();
+  ~CAppParamParserWebOS();
+
+protected:
+  void ParseArg(const std::string& arg) override;
+
+private:
+  int m_nArgs{0};
```

Comment:
What the point of this member?

Reply:
We count the args so that we only ignore if `m_nArgs` is 0 (because the json args from the gui launcher will always be the first args) and it starts with a `{` and ends with a `}`